lib/analytics.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import httplib;
+import json;
+
+class Analytics():
+
+    def __init__(self, domain):
+        self.domain = domain;
+        self.service = 'heroku-buildpack-typekit';
+        self.endpoint = 'jedkirby-develop-pr-51.herokuapp.com';
```

Comment:
This will ultimatley be the correct domain name; `api.jedkirby.com`

Reply:
Can you pull this from config?

Reply:
I could probably do that, but, not sure on the benefits..